addons/mail/static/src/components/composer/composer.scss

```diff
@@ -170,25 +180,10 @@
 
 .o_Composer {
     background-color: lighten(gray('300'), 7%);
+}
 
-    &.o-has-current-partner-avatar {
-
-        .o_Composer_actionButtons.o-composer-is-compact > *:last-child {
-            border-radius: 0 10px 10px 0;
-        }
-
-        .o_Composer_textInput {
-            border-radius: 10px 0 0 10px;
-
-            > .note-editor {
-                border-radius: 10px 0 0 10px;
-
-                > .note-editing-area {
-                    border-radius: 10px 0 0 10px;
-                }
-            }
-        }
-    }
+.o_Composer_actionButton-last.o-has-current-partner-avatar.o-composer-is-compact {
```

Comment:
Shouldn't it be either a modifier `o-last` or an actual element name `o_Composer_lastActionButton`?
Mixing the 2 in the name feels strange.

Reply:
@alexkuhn adviced to not use `o-last` as it could bring some extra style not wished (if I understood well).
The other 2 names are fine by me so, as you wish

Reply:
Feels more natural as a modifier, so `o-last`. But one reason to not use such a generic modifier is when there are global SCSS rules that apply on that, so workaround is to define modifier in super-long format `o_Composer_actionButton-last`.

Not sure if there are already defined and conflicting rulesets on `o-last`. Just assumed it was very likely, hence suggestion to use super-long format.

Reply:
Didn't find anything with `o-last` so I changed this into `o-last`